docs/src/architecture/accounts.md

```diff
@@ -52,6 +52,27 @@ Interface for accounts. In Miden every account is a smart contract. It has an in
 
 *Note: Since code in Miden is expresed as MAST, every function is a commitment to the underlying code. The code cannot change unnoticed to the user because its hash would change. Behind any MAST root there can only be `256` functions*
 
+#### Example Account Code
+Currently, Miden provides two standard implementations for account code. 
+
+A basic user account exposes the following code as its interface.
```

Comment:
I would create two sections: 
- **Basic user account** 
- **Basic fungible faucet**

Reply:
I changed it. Maybe the code of the faucet is too long for the docs. Let's see